examples/ase/ase_basic_example.ipynb

```diff
@@ -430,6 +434,33 @@
   {
    "cell_type": "code",
    "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "MissingDataError",
+     "evalue": "'\\'No value with the key \"particle.count\".\\''",
```

Comment:
Currently there's no `particle_count` value field in the FrameData: I'm not sure if there are other keys that should be being populated that are currently not populated...

Reply:
Ignore this, this was because we were accessing `latest_frame` rather than `current_frame`: after changing this everything works just fine!